### PR TITLE
work is complete

### PR DESCRIPTION
**Pull request by SolivanLau**
Work completed:
- created a new html file
- added some web copy

### CODE REVIEW

File: pr.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+	</head>
+	<body>
+		<h1>Helo frands</h1>
```
Comment (maxharrisnet) [inline pr.html]: Typo here, please fix 😉

**Review by maxharrisnet — CHANGES_REQUESTED** [review]

File: pr.html (hunk 1 above)
Comment (tingjc) [reply pr.html]: Please check spelling. git gud. uninstall

**Review by tingjc — CHANGES_REQUESTED** [review]
please fix typos

File: pr.html (hunk 1 above)
Comment (SolivanLau) [reply pr.html]: Sounds guuut

**Comment by SolivanLau** [comment]
Hi @maxharrisnet @tingjc Changes have been made, please give it another review

File: pr.html (hunk 1 above)
Comment (maxharrisnet) [reply pr.html]: Looks good now, thanks @SolivanLau!

File: pr.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+	</head>
+	<body>
+		<h1>Hello friends</h1>
```
Comment (SolivanLau) [inline pr.html]: adjusted typos as requested

File: pr.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+		<p>
+			Lorem ipsum, dolor sit amet consectetur adipisicing elit. Enim iure qui
```
Comment (SolivanLau) [inline pr.html:11]: added a p component

Comment (roo-tech) [inline pr.html:11]: why is the webcopy not available?

File: pr.html (hunk 2 above)
Comment (CosmicStarfire) [inline pr.html]: Add a class to this h tag because the styles are different.

**Review by CosmicStarfire — CHANGES_REQUESTED** [review]
request changes

File: pr.html (hunk 2 above)
Comment (SolivanLau) [reply pr.html]: Okay changes have been made, take a look at some comments

File: pr.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+	</head>
+	<body>
+		<h1 class="h5-styles">Hello friends</h1>
```
Comment (SolivanLau) [inline pr.html:9]: this is for miranda's request

Comment (SolivanLau) [reply pr.html:9]: @CosmicStarfire

**Pull request closed without merging**